File: FizzBuzz.c
/*
 *
 * FizzBuzz.c - FizzBuzz
 *
 * (c) gdifiore 2017  
 *      
 */

#include <stdio.h>

void FizzBuzz(int);

int main(void) {
    for (int i = 0; i <= 100; i++) {
        FizzBuzz(i);
    }

    return 0;
}

void FizzBuzz(int num) {
    if (num % 3 == 0 && num % 5 == 0) {
        puts("Fizzbuzz");
    } else if (num % 3 == 0) {
        puts("Fizz");
    } else if (num % 5 == 0) {
        puts("Buzz");
    } else {
        printf("%d\n", num);
    }
}

File: FizzBuzz.py
#!/usr/bin/env python

##########################
# FizzBuzz.py - FizzBuzz #
# (c) gdifiore 2017      #
##########################

def FizzBuzz(num):
    if num % 15 == 0:
        print("FizzBuzz")
    elif num % 5 == 0:
        print("Buzz")
    elif num % 3 == 0:
        print("Fizz")
    else:
        print(num)

for i in range(1, 101):
    FizzBuzz(i)